text/levenshtein_distance.ts

```diff
@@ -4,20 +4,23 @@ const { ceil } = Math;
 
 // This implements Myers' bit-vector algorithm as described here:
 // https://dl.acm.org/doi/pdf/10.1145/316542.316550
-const peq = new Uint32Array(0x10000);
+const peq = new Uint32Array(0x10ffff);
```

Comment:
```suggestion
const peq = new Uint32Array(0x110000);
```

Maybe add a test for `levenshteinDistance("\u{10FFFF}", "\u{10FFFF}")`?